lib/Rest/HttpRequest.cpp

```diff
@@ -880,7 +881,9 @@ VPackSlice HttpRequest::payload(VPackOptions const* options) {
   if ((_contentType == ContentType::UNSET) || (_contentType == ContentType::JSON)) {
     if (!_payload.empty()) {
       if (!_vpackBuilder) {
-        VPackParser parser(options);
+        VPackOptions validationOptions = *options;  // intentional copy
+        validationOptions.validateUtf8Strings = true;
+        VPackParser parser(&validationOptions); // adding the validatin here has no effect
```

Comment:
If the comment is correct, then I don't understand why we are setting the `validateUtf8Strings = true` here :thinking: